plugins/utils/mini.nix

```diff
@@ -82,9 +85,13 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
   callSetup = false;
   hasSettings = false;
   extraConfig = cfg: {
-    extraConfigLua = lib.foldlAttrs (lines: name: config: ''
-      ${lines}
-      require(${lib.nixvim.toLuaObject "mini.${name}"}).setup(${lib.nixvim.toLuaObject config})
-    '') "" cfg.modules;
+    extraConfigLua =
+      lib.foldlAttrs (lines: name: config: ''
+        ${lines}
+        require(${lib.nixvim.toLuaObject "mini.${name}"}).setup(${lib.nixvim.toLuaObject config})
+      '') "" cfg.modules
+      + lib.optionalString ((lib.hasAttr "icons" cfg.modules) && cfg.mockDevIcons) ''
```

Comment:
Nit: I think it is better to use language syntax when we don't need a (e.g. partially applied) function?

`(cfg.modules ? icons)`